<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>545</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="Static_Text">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="Variable_Text"/>
    </item>
    <item>
     <widget class="QTextEdit" name="Info_Text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Generate_Bt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>生成文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Copy_Pre_Bt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>前一个</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Copy_Next_Bt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>后一个</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="Template_comboBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;经典宋体简&quot;;</string>
   </property>
   <property name="text">
    <string>模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_Template_Bt">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>保存模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete_Template_Bt">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>删除模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Select_Template_Bt">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>选取模板</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
